### ISSUE TITLE
Send message onOpen

### ISSUE THREAD

**Issue by yantakus** [giantmachines/redux-websocket#19]
How do I do this with `redux-websocket`:

```javascript
w.on('open', () => w.send(
  JSON.stringify({ 
    event: 'subscribe', 
    channel: 'ticker', 
    symbol: 'tBTCUSD' 
  })
))
```

**Comment by nwestbury** [comment]
Wondering this exact thing. Worst comes to worst, it may be possible to write our own `websocket` middleware.

**Comment by yantakus** [comment]
@nwestbury I did it like this: https://github.com/yantakus/bfx/blob/a3bf5b26cfed24e4dd3887ecdd9a331910cf0d71/src/components/Ticker/index.js#L8-L17
but of course it's kinda hacky stuff comparing to `w.on('open', () => w.send(...`.

**Comment by nwestbury** [comment]
@yantakus Thanks, I ended up taking that approach, update the state on `WEBSOCKET_OPEN` and check the prop change.

**Comment by brianmcallister** [comment]
@yantakus You could do this now by tracking the state of your connection in the store, and then dispatching a `WEBSOCKET_SEND` action with your payload when the store changes, which is exactly what @yantakus did (which I don't think is hacky at all, you could find a lot of value tracking your connection state in your store).

We're working on overhauling this library to incorporate some of the feature requests we've gotten, clean up some tech debt, and make our documentation much, much better.

I've created a GitHub project to track progress, and you can follow along here: https://github.com/giantmachines/redux-websocket/projects/1

I've made a card in the backlog to track this issue, which you can see here: https://github.com/giantmachines/redux-websocket/projects/1#card-17569988

Because that card is there, I'm going to go ahead and close this issue, but please feel free to open up more issues if you have questions.

**Comment by brianmcallister** [comment]
Just by way of an update, we have added an `onOpen` callback into the configuration options of the middleware creator branch.

It's not yet published (though it is merged into `1.0-dev`), so you can check it out if you clone this repo.

Feel free to submit more feature requests in the future. We're getting very close to getting `1.0` released with some fancy new documentation as well.

**Comment by K12AZ** [comment]
how to use onOpen, Couldn't find the proper way,

COuld you able to explain?

TIA
